docs/zk-stack/running-a-hyperchain/locally.md

```diff
@@ -27,20 +33,130 @@ zk
 
 4. Last, start the wizard and follow instructions to set up and deploy your new hyperchain by running `zk stack init`
 
-a. Initially you want to `Configure new chain`
-
-b. Give it a name and chain id.
+   - Initially you want to `Configure new chain`
 
-c. Select localhost (default `matterlabs/geth`) and follow the wizard.
+   - Give it a name and chain id.
 
-- If you are doing this for the first time, several components need to be compiled/built, so do not worry if it takes a few minutes. The console will show what is going on anyways.
+   - Select localhost (default `matterlabs/geth`) and follow the wizard.
 
-d. If you don't want to configure any values for now and just want check the build process for a hyperchain, try out the `zk stack demo` command.
+     - If you are doing this for the first time, several components need to be compiled/built, so do not worry if it takes a few minutes. The console will show what is going on anyways.
 
-::: warning
+   - If you don't want to configure any values for now and just want check the build process for a hyperchain, try out the `zk stack demo` command.
 
-The commands above are not just running docker containers, but are actually building the code from the repo to spin up your hyperchain. For this reason the process might take some time. If you just want to run docker containers to play around with a zkSync chain, you can use zksync-cli dev. [Learn more here](../../build/tooling/zksync-cli/getting-started.md).
+:::warning
 
+The commands above are not just running docker containers, but are actually building the code from the repo to spin up your hyperchain. For this reason the process might take some time. If you just want to run docker containers to play around with a zkSync chain, you can use zksync-cli dev. Learn more here.
 :::
 
 Your hyperchain is now deployed to the base chain (most likely a local geth docker container) and configured. You can find all configuration in a new `.env` file created on `<project root>/etc/env/<your_chain_name_slug>.env`, and if you deployed test tokens, their addresses will be available at `<project root>/etc/tokens/<the_l1_identifier>.json`
+
+1. The wizard allows you to run the server in the end. If you chose not to, you’re still able to run it by executing
+
+```bash
+zk server --components "http_api,eth,state_keeper,housekeeper"
+```
+
+2. You can now run transactions and start playing with your hyperchain by using the RPC available at<http://localhost:3050>.
+
+   - Don't forget to deposit some ETH and fund your accounts on your hyperchain. To do so follow these instructions ->
```

Comment:
Is there content missing to the right of the arrow?

Reply:
Perhaps it's just confusing to see these arrows at the end of sentences

Reply:
This came from weird formatting on the quickstart. Updated to be hyperlink. Alternatively can also just remove and make "Follow the instructions below".